### PR TITLE
win32: Make win32_config macro switch:WITH_SSL,WITH_SASL_SCRAM work

### PR DESCRIPTION
**Pull request by prehistoric-penguin**

### CODE REVIEW

**Comment by prehistoric-penguin** [comment]
doozer/target/stretch-mips

File: src/rdgz.c [hunk 1]
```diff
@@ -122,3 +122,4 @@ void *rd_gz_decompress (const void *compressed, int compressed_len,
 		free(decompressed);
 	return NULL;
 }
+#endif
```
Comment (edenhill) [inline src/rdgz.c]: Add `#endif /* WITH_ZLIB `

**Comment by prehistoric-penguin** [comment]
Fixed.

File: src/rdkafka_sasl_scram.c [hunk 2]
```diff
@@ -900,3 +901,4 @@ const struct rd_kafka_sasl_provider rd_kafka_sasl_scram_provider = {
         .close         = rd_kafka_sasl_scram_close,
         .conf_validate = rd_kafka_sasl_scram_conf_validate,
 };
+#endif
```
Comment (edenhill) [inline src/rdkafka_sasl_scram.c]: missed one

File: src/rdgz.h [hunk 3]
```diff
@@ -41,5 +41,5 @@
  */
 void *rd_gz_decompress (const void *compressed, int compressed_len,
 			uint64_t *decompressed_lenp);
-
+#endif
```
Comment (prehistoric-penguin) [inline src/rdgz.h]: What about this line ? 
Need to be `endif /* WITH_ZLIB */` ?

Comment (edenhill) [reply src/rdgz.h]: yes of course

Comment (prehistoric-penguin) [reply src/rdgz.h]: Done!

**Comment by prehistoric-penguin** [comment]
doozer/target/stretch-mips

**Comment by edenhill** [comment]
Thank you for your effort but we'll need to decline this submission. see previous comment

**Pull request closed without merging**